test/specs/cli.js

```diff
@@ -181,7 +181,7 @@ describe('cli', function() {
             config: 'test/data/cli/cli.json'
         });
         return result.promise.then(function() {
-            assert(result.checker.getProcessedConfig().requireCurlyBraces);
+            expect(!!result.checker.getProcessedConfig().requireCurlyBraces).to.equal(true);
```

Comment:
It was converted automatically (there are around 100 cases like this) and `to have property` does not match for most. Quite often you have that property, but with negative value.